subsys/testsuite/ztest/Kconfig

```diff
@@ -79,6 +79,14 @@ config ZTEST_ASSERT_HOOK
 	  error test case. Remember to add ignore_fault tag in yaml file when
 	  using twister to run testing.
 
+config ZTEST_NO_YIELD
```

Comment:
this shall be default y, we can not afford to explore all test  applications to add them platform by platform.